src/App.php

```diff
@@ -1186,21 +1151,4 @@ private function outputResponseJson($data): void
         $this->setResponseHeader('Content-Type', 'application/json');
         $this->outputResponse($data);
     }
-
-    /**
-     * Generated HTML and JS for portal view registered to app.
-     */
-    private function getRenderedPortals(): array
-    {
-        // prevent looping (calling App::terminateJson() recursively) if JsReload is used in Modal
-        unset($_GET['__atk_reload']);
```

Comment:
This PR does not replace this unset in any way. It is either not needed or untested - if so, please submit an issue/testcase to fix it in the future.